wip-0002.md

```diff
@@ -0,0 +1,212 @@
+
+**Missing reveals**. All committers for which no valid reveal transaction exists in the block chain prior to the tally transaction itself must be listed in the`slashed_witnessess` field. To this effect, for a reveal transaction to be considered valid, it must be properly constructed such that it can be decoded into a compliant RADON type or RADON error.
+
+**Tally evaluation.** The rules for deciding whether committers which have produced valid reveal transactions for the referred request such that they exist prior to the tally transaction itself (revealers) belong to the `output` or `slashed_identities` fields remain the same as they were before this proposal.
+
+**Change output.** Given a request such that (1) it had insufficient commits, or (2) its count of committers outnumbers that of revealers, or (3) its count of slashed identies is not zero, or (4) any combination of 1, 2 and 3, a first `ValueTransferOutput` entry (change output) must exist in `outputs` such that it is spendable by the requestor, and its value equals the total value that was expected to be used for the `commit_fee` of the missing commits, plus the total value that was expected to be used for the `reward` and `reveal_fee` of the missing reveals, plus all the value that that was expected to be used as the `reward` of the slashed identities:
+
+    change = (reward + commit_fee) * (witnesses - committers_count) +
+             (reward + reveal_fee) * (witnesses - revealers_count) +
+             reward * slashed_count
+
+**All outputs are equal in value.** All the `ValueTransferOutput`  entries in `outputs` must have the same value, with the only exception of the change output.
+
+**Value of the outputs.** The value of every each of the `ValueTransferOutput` entries in the `outputs` field—except the change output—must equal to the result of adding the `reward` and `collateral` as specified in the data request, plus the result of the integer division of all the collateralized value lost by the identities found in `slashed_identities` by the number of rewarded identities:
+
+    output_value = reward + collateral * (1 + slashed_count / rewarded_count)
+
+**Collateral remainders.** May the integer division of all the collateralized value lost by the identities in `slashed_identities` by the number of rewarded identities leave a remainder, such amount will be implicitly considered part of the miner claimable fee, `tally_fee`.
+
+**No value is created or destroyed.** The aggregated value of all `ValueTransferOutput` entries in `output` must equal the total value of the `DataRequestOutput` from the data request, plus the total `collateral` value, minus the value of any fees claimed by miners.
+
+    Σoutput = dro_value +
+              collateral * witnesses -
+              commit_fee * (witnesses - committers_count) -
+              reveal_fee * (witnesses - revealers_count) - 
+              tally_fee
```

Comment:
With the fees claimed by minners will be:
    Σoutput = dro_value +
              collateral * witnesses -
              commit_fee * committers_count -
              reveal_fee * revealers_count - 
              tally_fee

Reply:
True, I rather used there the fees **not** claimed the miners, but that is totally wrong. I'm changing it now.

Reply:
Done. I used your exact formula. Resolving!